CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ option(PYTHON_MODULE "compile the Catima python module(requires numpy and cython
 option(TESTS "build tests" OFF)
 option(EXAMPLES "build examples" OFF)
 option(APPS "build catima applications" ON)
-option(GLOBAL "build with global, sources are required" OFF)
```

Comment:
GLOBAL is optional library to calculate ion charge states, this must be an option for build

Reply:
Ah OK, I removed this as I thought it did something else!